### PR TITLE
Lock mysql version temporarily

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -4,3 +4,6 @@
 default['osl-mysql']['xtrabackuprb']['version'] = '0.0.9'
 default['osl-mysql']['backup_dir'] = '/data/backup'
 default['osl-mysql']['enable_percona_client'] = false
+
+# Lock mysql version until upstream is fixed
+default['percona']['version'] = '5.7' if node['platform_version'].to_i > 7
```
Comment (ramereth) [inline attributes/default.rb:9]: Please look at the default recipe..this isn't the correct way to do it. Also you'll need to update the tests.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by ramereth** [comment]
@TheREK3R please fix the InSpec tests for the `client-percona-centos-8` suite? You might want to verify all of the other suites are working properly with EL8 as well.

**Comment by ramereth** [comment]
@TheREK3R can you please rebase this against current master branch and also squash the commits? Please also fix the new cookstyle issus.

**Review by ramereth — APPROVED** [review]
Please do a patch bump.

**Comment by TheREK3R** [comment]
!bump patch *

**Comment by TheREK3R** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v3.0.1.  Have a nice day!

**Pull request merged**